test/fixtures/util/util-inspect-error-cause.snapshot

```diff
@@ -0,0 +1,181 @@
+    [90m    at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main**)[39m
+    [90m    at node:internal*main*run_main_module**[39m
+  }
+}
+Error: Stack causes
+    at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+[90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+[90m    ... 4 lines matching cause stack trace ...[39m
+[90m    at node:internal*main*run_main_module**[39m {
+  [cause]: FoobarError: Individual message
+      at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+  [90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+  [90m    ... 4 lines matching cause stack trace ...[39m
+  [90m    at node:internal*main*run_main_module**[39m {
+    status: [32m'Feeling good'[39m,
+    extraProperties: [32m'Yes!'[39m,
+    [cause]: TypeError: Inner error
+        at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+    [90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+    [90m    at Module._extensions..js (node:internal*modules*cjs*loader**)[39m
+    [90m    at Module.load (node:internal*modules*cjs*loader**)[39m
+    [90m    at Module._load (node:internal*modules*cjs*loader**)[39m
+    [90m    at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main**)[39m
+    [90m    at node:internal*main*run_main_module**[39m
+  }
+}
+RangeError: New Stack Frames
+    at *
+[90m    at process.processTicksAndRejections (node:internal*process*task_queues**)[39m {
+  [cause]: Error: Stack causes
+      at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+  [90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+  [90m    ... 4 lines matching cause stack trace ...[39m
+  [90m    at node:internal*main*run_main_module**[39m {
+    [cause]: FoobarError: Individual message
+        at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+    [90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+    [90m    ... 4 lines matching cause stack trace ...[39m
+    [90m    at node:internal*main*run_main_module**[39m {
+      status: [32m'Feeling good'[39m,
+      extraProperties: [32m'Yes!'[39m,
+      [cause]: TypeError: Inner error
+          at Object.<anonymous> [90m(*[39mtest*fixtures*util*util-inspect-error-cause.js**[90m)[39m
+      [90m    at Module._compile (node:internal*modules*cjs*loader**)[39m
+      [90m    at Module._extensions..js (node:internal*modules*cjs*loader**)[39m
+      [90m    at Module.load (node:internal*modules*cjs*loader**)[39m
+      [90m    at Module._load (node:internal*modules*cjs*loader**)[39m
+      [90m    at Function.executeUserEntryPoint [as runMain] (node:internal*modules*run_main**)[39m
+      [90m    at node:internal*main*run_main_module**[39m
```

Comment:
We don’t want these internals in the snapshots. Renaming a method within the module loader shouldn’t break this unrelated test.

Reply:
Hey @GeoffreyBooth, I have a question. I'm making my edits in this PR after realizing I was going about it the wrong way. I'm having a hard time knowing when it's okay to edit the snapshot in order to get the tests to pass. I'm using past commits as a reference. But my question is, for the following, can I edit the snapshot to take "*" or should I edit my JS file to replace "*" with "90m" and "39m"?
```
      'RangeError: New Stack Frames\n' +
      '    at *\n' +
  +   '*[*m    at  *[*m {\n' +
  -   '*[90m    at *[39m {\n' +
```

Reply:
You should not edit the snapshots manually, use `NODE_REGENERATE_SNAPSHOTS=1` for regenerating snapshots.
you can probably remove colors from the snapshot
`str = str.replace(/[^\x00-\x7F]/g, '').replace(/\u001b\[\d+m/g, '')`

Reply:
@MoLow The linter don't like the control characters. Should I change the rules in the linter?

```
  22:12  error  Unexpected control character(s) in regular expression: \x00  no-control-regex
  23:12  error  Unexpected control character(s) in regular expression: \x1b  no-control-regex

✖ 2 problems (2 errors, 0 warnings)
```